scripts/infra

```diff
@@ -55,7 +55,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
         case "${1}" in
             plan)
-                update_batch_definitions
+                #update_batch_definitions
```

Comment:
I ran through the steps of a staging deploy through `infra play` and got "BATCH_ANALYSIS_JOB_NAME_REVISION: unbound variable", but I guess that's expected for now, and the workaround is just to comment this back in when doing a manual deploy (e.g. to production).